### ISSUE TITLE
Cover - Positioning slider works in reverse

### ISSUE THREAD

**Issue by kspijkers** [rospogrigio/localtuya#361]
I've got 2 cover switches type Loratap sc400w-eu working in the Local Tuya integration. I've put Positioning mode on 'timed' with the corresponding opening time in secs. for each cover. 

Now my positioning sliders for both switches are working in reverse. So value _100_ is fully opened and value _0_ is fully closed. The checkbox to invert 0-100 position works for position mode only, as stated in the options menu. 

Is it possible to add the invert option also available for the timed positioning mode?

![image](https://user-images.githubusercontent.com/73669259/108520640-34852280-72cb-11eb-8e45-c1435dd10c0f.png)

![image](https://user-images.githubusercontent.com/73669259/108520709-436bd500-72cb-11eb-8ac3-bf7114d1dcf0.png)

**Comment by dingausmwald** [comment]
Odd for me too, since windows meant to be clear as default, this should be 0% (0% extended blinds). But for other systems (google home, amazon?) 100% is open when im not mistaking. Smartlife app it is 0% = open.

A option would be nice, but then you have to have an option to change buttons too. I have both cover types, controlled by time and position. For my position ones, default is 0=open and 100=close. But the buttons are upside down. When i tick the checkbox to reverse, everything changes, including the buttons.
So my only solution is 100% open and 0% close.

**Comment by oxelwhy** [comment]
> Odd for me too, since windows meant to be clear as default, this should be 0% (0% extended blinds). But for other systems (google home, amazon?) 100% is open when im not mistaking. Smartlife app it is 0% = open.
> 
> A option would be nice, but then you have to have an option to change buttons too. I have both cover types, controlled by time and position. For my position ones, default is 0=open and 100=close. But the buttons are upside down. When i tick the checkbox to reverse, everything changes, including the buttons.
> So my only solution is 100% open and 0% close.

Yes, same and when the curtain automatically stop at open. The display will be hidden.

![image](https://user-images.githubusercontent.com/45725183/114861500-ee0df900-9e1f-11eb-94ce-e11c74461616.png)

**Comment by dingausmwald** [comment]
This just happens for a short time for me. In the period between home assistant is at 0% already but the curtain isnt. Maybe im mistakin, but you should setup the runtime of your curtains (via smartlife app, or button schema on device). Maybe your position field isnt setup right. For me, current position was 3, and set position was 2.

**Comment by oxelwhy** [comment]
Ya, same. My settings current position is 3 and set position is 2. On Tuya app, the 0% meaning open and 100% close (exactly same as the percentage set in this localtuya). The only thing is that when the 0% position, it is grey out as shown on picture, but if I click the stop button, the arrow up is lighted up. 

I see that when blinds actually open ( the localtuya see it as closed, that the reason the arrow up became clickable) and vice version. Perhaps can just invert the logic for this?

**Comment by WarlaxZ** [comment]
use position and click the invert position button
